Mirror/Runtime/ClientScene.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
 using UnityEngine;
+using Guid = System.Guid;
```

Comment:
Why not 'using System;' here?

Reply:
because then you get:
"Object is ambiguous"  error in this line:

```                Object.Destroy(s_ReadyConnection.playerController.gameObject);```

Reply:
<img width="970" alt="screen shot 2018-10-16 at 1 47 10 pm" src="https://user-images.githubusercontent.com/466007/47039645-09fa8c00-d14a-11e8-986d-36ae5ed75ee6.png">